File: JAVA/level1/서울에서_김서방_찾기.java
class Solution {
  public String solution(String[] seoul) {
      String answer = "";

      for(int i=0; i<seoul.length; i++)
          if(seoul[i].compareTo("Kim") == 0)
              return "김서방은 " + Integer.toString(i) + "에 있다";

      return answer;
  }
}

File: C++/level1/서울에서_김서방_찾기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<string> seoul) {
    string answer;
    vector<string>:: iterator it = find(seoul.begin(), seoul.end(), "Kim");

    answer = "김서방은 " + to_string(distance(seoul.begin(), it)) + "에 있다";
    return answer;
}
